### PR TITLE
ACMS-000: Release Acquia CMS Starter kit & remove acquia/acquia_cms.

### PR DESCRIPTION
**Pull request by vishalkhode1**
- Removed [acquia/acquia_cms](https://github.com/acquia/acquia_cms) in favor of latest stable release of Acquia CMS Starter kit.
- Updated README.md file accordingly.

### CODE REVIEW

File: README.md
Comment (balsama) [inline README.md:8]: Let's link this to the project like the others

File: README.md
Comment (balsama) [inline README.md:14]: Link these as well (where applicable)

File: README.md
Comment (balsama) [inline README.md:19]: This copy seems awkward. It would be better if I could scan the list for the names of each.
```suggestion
* **Low Code** for low code CMS with Acquia Site Studio.
* **Headless** for CMS configurations in micro services environments.
* **Community** for more minimal starting points allowing greater customization.
```
It's also a confusing list to me. Both 1 & 3 seem to be "minimal". What do I choose for a "normal"/"standard" install? Also, what about the third option is "community" focused? Shouldn't it just be called "Minimal"?

File: composer.json
Comment (balsama) [inline composer.json:14]: The `.0` is superfluous.
```suggestion
        "acquia/acquia-cms-starterkit": "^1",
```

File: composer.json
Comment (balsama) [inline composer.json:131]: This package exists on the main packagist:
https://packagist.org/packages/acquia/acquia-cms-starterkit

We should get the administration of that sorted out (e.g. it's not currently auto-updated). This repo definition should be removed.

**Review by balsama — CHANGES_REQUESTED** [review]

**Comment by balsama** [comment]
I also requested reviews from @secretsayan (maintainer of ORCA) and @danepowell (maintainer of this project).

File: README.md
Comment (nagwani) [reply README.md:19]: Currently Community/Minimal is Normal.  This has been named community after a lot of internal discussions. So we have low code which is enterprise, Community and Headless.

**Comment by danepowell** [comment]
As the project maintainer, I'm generally okay with this (haven't had time to thoroughly review).

But we need approval from @TravisCarden and/or @secretsayan since we expected this to break ORCA. Surprisingly, ORCA tests are still passing on this PR and I'm not sure how.

File: README.md
Comment (vishalkhode1) [reply README.md]: Done.

File: README.md
Comment (vishalkhode1) [reply README.md]: Updated the README with better description.

File: composer.json
Comment (vishalkhode1) [reply composer.json]: Done.

File: composer.json
Comment (vishalkhode1) [reply composer.json]: Done.

File: README.md
Comment (balsama) [inline README.md:13]: Any reason we can't link this one too?
https://github.com/acquia/memcache-settings

**Review by balsama — APPROVED** [review]
Small nit about linking `/acquia/memcache-settings`. Otherwise looks good. 
I approve pending a review from the ORCA maintainers.

File: README.md
Comment (vishalkhode1) [reply README.md]: Thanks @balsama. I've made the changes. :)

**Review by TravisCarden — APPROVED** [review]
I think the impact this will have on ORCA is that it just won't install Acquia CMS anymore. In effect, every build will become an isolated build until we modify ORCA to actually use the starterkit during fixture creation. In other words, I don't think this will break anything, we'll just lose functionality--which I believe we agreed we're OK with for now. You have my (slightly anxious) approval. 😉

**Pull request merged**
